docs/ibc/light-clients/genesis.md

```diff
@@ -1,3 +1,35 @@
 <!--
 order: 10
--->
\ No newline at end of file
+-->
+
+# Genesis Metadata
+
+Learn how to implement the `ExportMetadata` interface {synopsis}
+
+## Pre-requisites Readings
+
+- [Cosmos SDK module genesis](https://docs.cosmos.network/v0.47/building-modules/genesis) {prereq}
+
+`ClientState` instances are provided their own isolated and namespaced client store upon initialisation. `ClientState` implementations may choose to store any amount of arbitrary metadata in order to perform light client operations correctly. 
```

Comment:
mega nit, but maybe a more precise wording for "perform light client operations" is "verify the counterparty consensus state"?

Reply:
I changed to:

> in order to verify counterparty consensus state and perform light client updates correctly